src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,50 @@
 import React from 'react';
+import { Movie } from '../Movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type Props = {
+  movieFromServer: Omit<Movie, 'imdbId'>,
+};
+
+export const MovieCard: React.FC<Props> = ({ movieFromServer }) => {
+  const {
+    title,
+    description,
+    imgUrl,
+    imdbUrl,
+  } = movieFromServer;
+
+  return (
+    <div className="card">
+      <div className="card-image">
+        <figure className="image is-4by3">
+          <img
+            src={imgUrl}
+            alt="Film logo"
+          />
+        </figure>
+      </div>
+
+      <div className="card-content">
+        <div className="media">
+          <div className="media-left">
+            <figure className="image is-48x48">
+              <img src={imgUrl} alt="imdb" />
+            </figure>
```

Comment:
I think this place needs to use "images/imdb-logo.jpeg"

Reply:
yeah, thank you 👍